test/addons/async-hooks-promise/test.js

```diff
@@ -26,7 +26,7 @@ hook0.disable();
 
 let pwrap = null;
 const hook1 = async_hooks.createHook({
-  init(id, type, tid, resource) {
+  init(_id, _type, _tid, resource) {
```

Comment:
It's always possible to do like this to omit unused arguments: `init(...[,,, resource]) { ... }`